src/main/drivers/stm32/dshot_bitbang.c

```diff
@@ -504,84 +507,91 @@ static bool bbMotorConfig(IO_t io, uint8_t motorIndex, motorPwmProtocolTypes_e p
     return true;
 }
 
-static bool bbUpdateStart(void)
+static bool bbTelemetryWait(void)
+{
+    // Wait for telemetry reception to complete
+    bool telemetryPending;
+    bool telemetryWait = false;
+    const timeUs_t startTimeUs = micros();
```

Comment:
(`uint32_t` is large enough here)

Reply:
For what?

Reply:
`timeUs_t` may be defined as `uint64_t`. Using only `uint32_t` (and `cmp32`) will work fine for delays up to 35 minutes, which is enough here. And it will save few cycles

Reply:
I think it’s clearer to always use `timeUs_t` for us manipulation.

Reply:
IMO correct way it to always use 32bit `timeUs_t` and write code that handles overflow correctly. 64bit is overkill.